content/zh-cn/blog/advanced4.md

```
@@ -0,0 +1,230 @@
+这就带来了较大的配置和使用成本。
+
+
+
+<font style="color:#117CEE;">而 Koupleless 模块化设计虽然借鉴了 OSGI 的设计，却也简化了这一模块共享和通信的使用方式。</font>
+
+<font style="color:#117CEE;"></font>
+
+#### 1⃣️免配置的类与资源共享方式
+1. OSGI 模块角色都是对等的，在这一点上 Koupleless 与之不同。Koupleless 对模块做了区分，定义了两种不同角色：模块与基座（基座背后也是一个模块）。
+
+![](https://intranetproxy.alipay.com/skylark/lark/0/2024/png/149473/1722234767836-5acf6726-3b15-4fb4-b275-08de3e09f7ac.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)